src/Security/Authentication/Negotiate/src/Events/AuthenticatedContext.cs

```diff
@@ -13,9 +13,7 @@ public class AuthenticatedContext : ResultContext<NegotiateOptions>
         /// <summary>
         /// Creates a new <see cref="AuthenticatedContext"/>.
         /// </summary>
-        /// <param name="context"></param>
-        /// <param name="scheme"></param>
-        /// <param name="options"></param>
+        /// <inheritdoc />
```

Comment:
I didn't know you could inherit doc like this?  So what exactly is the result of mixing summary an inheritdoc?

Reply:
It picks the docs from the base type for the values that weren't supplied - in this case, the docs for the parameters.